File: SmartGardenBackEnd/src/main/java/be/pxl/Services/ICropService.java
package be.pxl.services;

import be.pxl.models.Crop;

/**
 * Created by 11308157 on 28/10/2016.
 */
public interface ICropService {
    Iterable<Crop> getAll();
    void createCrop(Crop crop);
    Crop getCropByName(String name);

    void deleteCrop(int cropID);
}

File: Smart_Garden_UWP/Smart_Garden_UWP/Services/Interfaces/ICropService.cs
using Smart_Garden_UWP_Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Smart_Garden_UWP.Services.Interfaces
{
    public interface ICropService
    {
        Task<Boolean> addCrop(Crop crop);

        Task<Boolean> deleteCrop(Crop crop);

        Task<List<Crop>> getAllCrops();

        Task<Crop> getCropByName(String name);
    }
}
